Doc/library/pdb.rst

```diff
@@ -76,6 +76,9 @@ at the location you want to break into the debugger.  You can then step through
 the code following this statement, and continue running without the debugger
 using the :pdbcmd:`continue` command.
 
+.. versionadded:: 3.7
+   The built-in :func:`breakpoint()`, when called with defaults, can be used instead of ``import pdb; pdb.set_trace()``.
```

Comment:
Minor feedback: line was too long, should be less than 80 characters.
```suggestion
   The built-in :func:`breakpoint()`, when called with defaults, can be used 
   instead of ``import pdb; pdb.set_trace()``.
```

Reply:
Are you sure that there is that limit for the documentation? Can you direct me to where it is?
I checked both the online documentation and the .chm file and neither complies with max 79 chars.
I know that limit exists for source code and the max 72 chars for docstrings and long blocks of comments, but I never saw any mention of 79 for the documentation.

Reply:
https://devguide.python.org/documenting/#use-of-whitespace
"The maximum line length is 80 characters for normal text,"

Reply:
Ok, thanks. Always learning something new everyday.